scripts/download-prebuilds.mjs

```diff
@@ -85,19 +89,38 @@ export async function downloadPrebuilds({verbose} = {verbose: false}) {
   });
 }
 
-/**
- * @param {{name: string, version: string, target: string}} opts
- * @returns {string}
- */
-function getArtifactTarballName({name, version, target}) {
-  return `${name}-${version}-node-${NODE_ABI}-${target}.tar.gz`;
+function getNodeJsMobileNodeVersions() {
+  const nodeVersionFilePath = new URL(
+    'android/libnode/include/node/node_version.h',
```

Comment:
this corresponds to the header file that the NodeJS project places several version-related constants in